prboom2/src/dsda/music.c

```diff
@@ -63,23 +48,12 @@ int dsda_GetDehMusicIndex(const char* key, size_t length) {
 
 int dsda_GetOriginalMusicIndex(const char* key) {
   int i;
-  const char* c;
 
   for (i = 1; deh_musicnames[i]; ++i)
     if (!strncasecmp(deh_musicnames[i], key, 6))
       return i;
 
   return -1;
-
-  // is it a number?
```

Comment:
This stays

Reply:
Commented out code should generally include an explanation of why it's expected to be restored in the future.